libraries/lib-audio-io/AudioIO.cpp

```
@@ -2599,19 +2610,28 @@ static void DoSoftwarePlaythrough(constSamplePtr inputBuffer,
                                   sampleFormat inputFormat,
                                   unsigned inputChannels,
                                   float *outputBuffer,
-                                  unsigned long len)
-{
-   for (unsigned int i=0; i < inputChannels; i++) {
+                                  unsigned long len,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'DoSoftwarePlaythrough' of convertible types are easily swapped by mistake

(bugprone-easily-swappable-parameters)